src/main/java/baritone/pathing/movement/MovementHelper.java

```diff
@@ -111,7 +105,7 @@ static boolean canWalkThrough(CalculationContext context, int x, int y, int z, I
             if (Baritone.settings().assumeWalkOnWater.get()) {
                 return false;
             }
-            IBlockState up = context.get(x, y + 1, z);
```

Comment:
I don't understand, why these changes?

Reply:
ok now you're speaking my language with words like "performance" referring to cost calculation. hell yeah now i agree. it make baritone like sanik.

Reply:
awesome style